exonum_launcher/supervisor.py

```diff
@@ -85,17 +85,25 @@ def create_deploy_request(self, artifact: Artifact, spec_loader: RuntimeSpecLoad
 
         return deploy_request.SerializeToString()
 
-    def create_start_instance_request(self, instance: Instance, config_loader: InstanceSpecLoader) -> bytes:
-        """Creates a start instance request for given instance."""
+    def create_start_instance_request(self, instances: List[Instance],
```

Comment:
Maybe rename to `create_start_instances_request`, since now there are many of them?

Reply:
Yes